### PR TITLE
Add information about oC hosted update server

### PR DESCRIPTION
**Pull request by michaelstingl**
@settermjd Please help me with the link, check rst formatting and perhaps you can improve the text style of this page in general.

### CODE REVIEW

File: branded_desktop_client/update_branded_desktop_clients.rst [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+
+The easy way is, to notify your ownCloud support about the location where
+you uploaded your new generated branded client binaries.
+
+You need to perform the following steps:
+
+#. Generate branded clients.
+#. Upload branded clients to your Web server.
+
+   -  Windows example:
```
Comment (settermjd) [inline branded_desktop_client/update_branded_desktop_clients.rst]: @michaelstingl I don't really understand what is meant by these examples. Can you clarify please?

**Review by settermjd — COMMENTED** [review]
In a local copy of the branch, I've made some modifications. Other than the question on the examples, the changes seem fine to me.

File: branded_desktop_client/update_branded_desktop_clients.rst (hunk 1 above)
Comment (michaelstingl) [inline branded_desktop_client/update_branded_desktop_clients.rst]: Customers build branded desktop sync clients using ownBrander. They need to make them available for download on their webserver. ownCloud supports then needs 1 URL for the Windows binaries, 3 URLs for the macOS binaries, and the URL of the page where users can download the new clients manually.

Comment (settermjd) [inline branded_desktop_client/update_branded_desktop_clients.rst]: Right. Ok. Thanks.

**Comment by settermjd** [comment]
@michaelstingl I've made some minor revisions to both the text and layout. If you're happy with them, then I'll merge them. Otherwise, feel free to.

**Comment by michaelstingl** [comment]
@settermjd This document describe 2 ways how to update branded clients:

1. Branded clients connect to updates.owncloud.com, ownCloud support team will configure it for customer
2. Customers host updater script on own infrastructure

That's why I had two main sections. Now it's not clear that are alternative ways.

Better have 2 separate pages to make this clear?

**Comment by settermjd** [comment]
@michaelstingl thanks for pointing that out. To be honest, it didn't come across that way to me; but then, I'm new to this repository.

**Comment by settermjd** [comment]
@michaelstingl I added the sub-section header back in again, and added a bit more intro text to the page as well. Does this do what you want?

**Pull request closed without merging**